src/JobFeed/index.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { MockedProvider } from '@apollo/client/testing';
+import JobFeed from './index';
+import { QUERY_JOBS } from '../graphql/queries';
+
+const mockedResponses = [{
+  request: { query: QUERY_JOBS },
+  result: {
+    data: {
+      jobs: [
```

Comment:
I recommend having multiple objects here that have different properties and Check if they are rendered, or just check for 2 out of 3 and check for the existence of 3 elements in the test